pages/use-solid.md

```diff
@@ -39,8 +39,9 @@ The following table lists some Pod Provider:
 | [Inrupt Pod Spaces](https://signup.pod.inrupt.com) | [Inrupt, Inc.](https://inrupt.com/terms-of-service)                        | [Amazon](https://aws.amazon.com)              | USA              |
 | [inrupt.net](https://inrupt.net)                   | [Inrupt, Inc.](https://inrupt.com/terms-of-service)                        | [Amazon](https://aws.amazon.com)              | USA              |
 | [solidcommunity.net](https://solidcommunity.net/)  | [Solid Project](https://github.com/solid/solidcommunity.net_operations)    | [Digital Ocean](https://www.digitalocean.com) | UK               |
-| [solidweb.org](https://solidweb.org)               | [Solid Grassroots](https://gitlab.com/groups/solidweb.org/-/group_members) | [Hosteurope](https://www.hosteurope.de)       | Germany          |
+| [solidweb.org (NSS)](https://solidweb.org)               | [Solid Grassroots](https://gitlab.com/groups/solidweb.org/-/group_members) | [Hosteurope](https://www.hosteurope.de)       | Germany          |
 | [trinpod.us](https://trinpod.us)                   | [Graphmetrix, Inc.](https://graphmetrix.com/terms)                         | [Amazon](https://aws.amazon.com)                                                              | USA             |
+| [solidweb.me (CSS)](https://solidweb.me)               | [Matthias Evering](https://github.com/ewingson) | [Hosteurope](https://www.hosteurope.de)       | Germany          |
```

Comment:
If we're going to specify the server software, I think it'd be more consistent to have a separate column for that. But really, it shouldn't matter for regular Solid users, since servers should (maybe it's my pipe dream? :P) be interoperable. It might be relevant to developers wishing to test that interoperability, but this page isn't focused on just developers... What do you think?

```suggestion
| [solidweb.org](https://solidweb.org)               | [Solid Grassroots](https://gitlab.com/groups/solidweb.org/-/group_members) | [Hosteurope](https://www.hosteurope.de)       | Germany          |
| [trinpod.us](https://trinpod.us)                   | [Graphmetrix, Inc.](https://graphmetrix.com/terms)                         | [Amazon](https://aws.amazon.com)                                                              | USA             |
| [solidweb.me](https://solidweb.me)               | [Matthias Evering](https://github.com/ewingson) | [Hosteurope](https://www.hosteurope.de)       | Germany          |
```